modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -541,8 +541,20 @@ protected function getEmailRecordFieldValue($field, $emailHeader, $inboundEmail,
                 $uid = $emailHeader['uid'];
                 $importedEmailBeans = BeanFactory::getBean('Emails');
                 $is_imported = $importedEmailBeans->get_full_list('',
-                    'emails.uid LIKE "' . $uid . '"');
-                if (count($is_imported) > 0) {
+                    'emails.uid LIKE "' . $uid . '"'); 
+                
+                if (null === $is_imported) {
+                    $is_imported = [];
+                }
```

Comment:
This might be not needed, if it is null it's not an instance of Countable and will be casted to an empty array just below.